### PR TITLE
Week 10 day 02 mod 02

### PR DESCRIPTION
**Pull request by cameronjacoby**

### CODE REVIEW

File: week-10-angular/day-02/module-02/readme.md [hunk 1]
```diff
@@ -1 +1,233 @@
...
+Two more that you will use often:
+
+* `ng-model`: binds a form element's value to a property on `$scope`.
+* `ng-repeat`: iterates over a collection.
+
+Here is a full <a href="https://docs.angularjs.org/api/ng/directive" target="_blank">list of Angular's built-in directives</a>.
+
+### ngModel
+
+Assume you have the following Angular app set up for a trainer to manage their Pokemon. Here is the HTML snippet containing your Angular app:
```
Comment (eerwitt) [inline week-10-angular/day-02/module-02/readme.md]: Pok**é**mon :)

File: week-10-angular/day-02/module-02/readme.md [hunk 2]
```diff
@@ -1 +1,233 @@
...
+```html
+<!-- index.html.erb -->
+
+<!DOCTYPE html>
+<html ng-app="pokemonApp">
+<head>
+  ...
+
+  <!-- angular -->
+  <script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/angular.js/1.4.4/angular.min.js"></script>
```
Comment (eerwitt) [inline week-10-angular/day-02/module-02/readme.md]: Might want to use 1.4.8?

**Comment by eerwitt** [comment]
:+1: looks great to me, only serious comment is using 1.4.8.

**Comment by cameronjacoby** [comment]
@eerwitt Good catch - thanks! I updated the version number and also fixed the Pokémon spelling :smile_cat:

**Pull request merged**